gradle/libs.versions.toml

```diff
@@ -1,10 +1,10 @@
 [versions]
-minSdk = "23"
+minSdk = "21"
 compileSdk = "34"
 kotlin = "1.9.10"
 agp = "8.1.0"
 androidx-compose-ui = "1.5.1"   # https://developer.android.com/jetpack/androidx/releases/compose-ui
-androidx-compose-ui-material3 = "1.1.1"
+androidx-compose-ui-material3 = "1.1.2"
```

Comment:
Can you undo this change? It's best to keep unrelated changes separate from a PR.

Reply:
Sure, I'll make a commit reverting this change tomorrow.

Reply:
The change is reversed.